draft-ietf-tls-tls13.md

```diff
@@ -2301,9 +2303,14 @@ EdDSA algorithms
 : Indicates a signature algorithm using EdDSA as defined in
   {{RFC8032}} or its successors. Note that these correspond to the
   "PureEdDSA" algorithms and not the "prehash" variants.
+
+ecdsa_sha1
```

Comment:
I agree with Martin's first point, but would probably benefit from a bit more context on the second point.